pingdom-checks/provider-directory.ping

```diff
@@ -1,8 +1,23 @@
 # -*- sh -*-
 
+PRODUCTION_PROVIDER_DIRECTORY_API_KEY=$(get-secret "/dvp/production/provider-directory/api-key")
 SANDBOX_PROVIDER_DIRECTORY_API_KEY=$(get-secret "/dvp/lab/provider-directory/api-key")
 PROVIDER_DIRECTORY_SLACK_ID="${SLACK_VASDVP_MONITORING},${SLACK_API_ALARMS_PROVIDER_DIRECTORY}"
 
+#
+# Production
+#
+
+pingdom save-check \
```

Comment:
I don't think this is available through the gateway:
```
$ curl "https://api.va.gov/services/provider-directory/v0/health" | jq
{
  "message": "no Route matched with those values"
}
```

Reply:
what would be the process for adding it?

Reply:
I was informed that Adam is currently tracking that down with Cassowary